### PR TITLE
Hotfix/changed link style

### PR DESCRIPTION
**Pull request by kumardennis**

### CODE REVIEW

File: src/templates/case-template.js [hunk 1]
```diff
@@ -42,7 +42,7 @@ const StyledHeader = styled.header`
     }
 `;
 const StyledMeta = styled.div`
-    // display: block;
+    // display: block;display
```
Comment (edanchenkov) [inline src/templates/case-template.js]: Why to commit this? Why not to remove this whole line instead?

File: src/templates/case-template.js [hunk 2]
```diff
@@ -109,7 +134,7 @@ const StyledBlogFooter = styled.div`
         color: white;
     }
     svg {
-        // margin-right: 1rem;
+        // margin-right: 1rem;margin-right
```
Comment (edanchenkov) [inline src/templates/case-template.js]: And here?

File: src/templates/case-template.js (hunk 1 above)
Comment (kumardennis) [reply src/templates/case-template.js]: i am not sure where this came from :/

Comment (kumardennis) [reply src/templates/case-template.js]: i will remove the commented code since its pointless anyway

File: src/templates/news-template.js [hunk 3]
```diff
@@ -67,7 +67,24 @@ const StyledPost = styled.div`
...
+        transition: color 0.5s;
+    }
+
+    a:visited {
+        color: #b5a1ff;
+    }
+
+    a:hover {
+        color: #ffffff;
+        cursor: pointer;
```
Comment (edanchenkov) [inline src/templates/news-template.js]: Is it not a pointer already?

**Review by edanchenkov — APPROVED** [review]

File: src/templates/case-template.js (hunk 2 above)
Comment (kumardennis) [reply src/templates/case-template.js]: its weird it happens when press cntrl+s to save the file :/

File: src/templates/news-template.js (hunk 3 above)
Comment (kumardennis) [reply src/templates/news-template.js]: i just copy pasted lol, but these small things will move to ```a``` instead psuedoclass, if its not already poiner

Comment (edanchenkov) [reply src/templates/news-template.js]: I believe it is `cursor: pointer;` by default unless overwritten previously for whatever reason. I guess it is pretty obviously that nothing should ever change this behaviour. So just remove it or if it breaks something, try to fix what's really cause it.

**Review by victorv-sbp — APPROVED** [review]

**Pull request merged**